src/runtime_src/core/pcie/driver/linux/xocl/subdev/mailbox.c

```diff
@@ -1311,6 +1311,8 @@ static bool chan_do_tx(struct mailbox_channel *ch)
 				do_sw_tx(ch);
 			else
 				do_hw_tx(ch);
+			/* reset timer */
+			msg_timer_on(ch->mbc_cur_msg, 0);
```

Comment:
Do we have any other locations that might need to do this step too?

Reply:
> Do we have any other locations that might need to do this step too?

I did not see any.